src/app.js

```diff
@@ -1,21 +1,35 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+const fs = require('fs');
+
+const PORT = process.env.PORT || 3000;
+
+const server = http.createServer((req, res) => {
+  const normalizedUrl = new URL(req.url, `http://${req.headers.host}`);
+  const pathnameArray = normalizedUrl.pathname
+    .split('/')
+    .filter(path => path !== '');
+
+  if (pathnameArray[0] !== 'file') {
+    res.end('Correct request is: "/file/<path_to_file>"');
+
+    return;
+  }
+
+  const filename = pathnameArray.slice(1).join('/') || 'index.html';
+
+  fs.readFile(`./src/public/${filename}`, (err, data) => {
+    if (!err) {
+      res.end(data);
+    }
+
+    res.statusCode = 404;
+    res.end();
```

Comment:
You should send a 'Not Found' message to the client when the file does not exist. Right now you just set the status code to 404 and end the response without any message.

Reply:
According to documentation, if i set response status code, status message will be set automatically. So, I don't need to do that.